packages/opentelemetry-plugin-express/src/types.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { kLayerPatched } from './express';
+import { Request } from 'express';
+import { PluginConfig, Attributes } from '@opentelemetry/types';
+
+export const _MIDDLEWARES_STORE_PROPERTY = '__ot_middlewares';
```

Comment:
This is a very undescriptive name with no documentation

Reply:
I will update it to explain the goal of that and its link to the http plugin.